Capfile

```diff
@@ -18,7 +18,7 @@ require 'capistrano/deploy'
 # require 'capistrano/chruby'
 require 'capistrano/bundler'
 # require 'capistrano/rails'
-require 'capistrano/rvm' # gdor-indexer needs jruby until merge-manager
+require 'capistrano/rvm' # gdor-indexer used to need jruby (for merged records using solrmarc-sw)
```

Comment:
i'm not sure we even need capistrano any more -- we use it as a gem, not as an app now, yes?